systemuicontroller/src/main/java/com/google/accompanist/systemuicontroller/SystemUiController.kt

```diff
@@ -104,26 +104,38 @@ interface SystemUiController {
      *
      * @return true, if API is 29+ and the system is ensuring contrast, false otherwise.
      */
-    fun isNavigationBarContrastEnforced(): Boolean = false
+    fun isNavigationBarContrastEnforced(): Boolean
 }
 
 /**
- * Remembers a [SystemUiController] which supports Android devices.
+ * Remembers a [SystemUiController] for the current device.
  */
 @Composable
-fun rememberAndroidSystemUiController(
+fun rememberSystemUiController(
     view: View = LocalView.current
```

Comment:
do you really expect anyone to pass their own Views? maybe it is too confusing to allow providing your View?